lib/cocoapods/project.rb

```diff
@@ -383,7 +381,7 @@ def group_for_path_in_group(absolute_pathname, group, reflect_file_system_struct
     # Returns the name to be used for a the variant group for a file at a given path.
     # The path must be localized (within an *.lproj directory).
     #
-    # @param  [Pathname] The localized path to get a variant group name for.
+    # @param  [Pathname] path The localized path to get a variant group name for.
```

Comment:
haha thank you - this has been bugging me but I never found a good time to update it